projects/core/src/lib/components/home-page/home-page.component.html

```diff
@@ -0,0 +1,51 @@
+<div class="home">
+	<section class="about">
+		<h2 class="home__title">What is JS Middle Developer Growth ?</h2>
```

Comment:
__title should be part of about block. In the block is not needed to contain the elements of another block. plese fix it everywhere

Reply:
i made it because title everywhere should have same styles, but if it is wrong, i will use @extend method in scss to avoid repetition

Reply:
done